lib/rules/selector-pseudo-class-no-unknown/index.js

```diff
@@ -84,9 +84,17 @@ const rule = function(actual, options) {
               return;
             }
 
-            if (pseudoNode.prev()) {
+            const firstSelectorNode =
+              pseudoNode.parent && pseudoNode.parent.nodes
```

Comment:
Should we check `parent` here? Be good to find use case where `pseudoNode` doesn't contain `parent` (if this use case exists).

Reply:
cc @jeddy3 thoughts?

Reply:
We can probably sneak this into 8.1.0 if no other chanfes to the PR are required

Reply:
> Should we check parent here?

Sure, no harm in guarding against it.